src/scenes/Species/SpeciesAddView.tsx

```diff
@@ -79,23 +76,23 @@ export default function SpeciesAddView({ reloadData }: SpeciesAddViewProps): JSX
           sx={{
             display: 'flex',
             margin: '0 auto',
-            width: isMobile || featureFlagMockedSpecies ? '100%' : '700px',
+            width: '100%',
             paddingLeft: theme.spacing(isMobile ? 0 : 4),
             paddingRight: theme.spacing(isMobile ? 0 : 4),
             paddingTop: theme.spacing(5),
           }}
         >
           <Grid
             container
-            width={isMobile || featureFlagMockedSpecies ? '100%' : '700px'}
+            width={'100%'}
```

Comment:
Same here: isMobile ? '100%' : '700px',

Reply:
I had to put these back, it seems the condition was correct in the first place and we do need it to be 100%. Since there are more fields, we need more real estate to show everything. Automated tests are failing because it can't click on the dropdown items.

[uploading 0f963eaf80aabcc130f2119974e4368e74c207eb.png...]